pkg/collector/osmlogs_collector.go

```diff
@@ -29,7 +29,7 @@ func NewOsmLogsCollector(exporter interfaces.Exporter) *OsmLogsCollector {
 // Collect implements the interface method
 func (collector *OsmLogsCollector) Collect() error {
 	// Get all OSM deployments in order to collect information for various resources across all meshes in the cluster
-	meshList, err := getResourceList("deployments", "app=osm-controller", "-o=jsonpath={..meshName}", " ")
+	meshList, err := getResourceList([]string{"get", "deployments", "--all-namespaces", "-l=app=osm-controller", "-o=jsonpath={..meshName}"}, " ")
```

Comment:
Nit: `-l` to `--selector` for parallelism to the groundTruthMap

Reply:
hm good catch. I think it'll make more sense to change everywhere to `-l` since we also use `-n` and `-o` everywhere else! Will make that change in a separate PR